addon/components/horizontal-bar-chart.js

```diff
@@ -237,13 +237,24 @@ const HorizontalBarChartComponent = ChartComponent.extend(FloatingTooltipMixin,
     };
   }),
 
+  /*
+   * Ensures that grouping labels will be on the left when all data is negative or 0
+   * @private
+   * @param {Object}
+   * @return {Boolean}
+   */
+  _isNotPositive: function(d) {
+    return ((d.value < 0) || ((d.value === 0) && this.get('hasAllNegativeValues')));
```

Comment:
Oh wait, never mind, misread this. You should pull out these into separate expressions:
```
if (d.value < 0) { 
  return true; 
}

if (d.value === 0 && this.get('hasAllNegativeValues')) {
  return true
}

return false;
```